src/renderer/actions/install-path.utils.tsx

```diff
@@ -3,16 +3,28 @@ import Store from 'electron-store';
 
 const settings = new Store;
 
-export async function setupInstallPath(): Promise<string> {
-    const currentPath = settings.get<string>('mainSettings.msfsPackagePath');
+export async function setupInstallPath(target: string): Promise<string> {
```

Comment:
I think it would be better to separate this into two functions

Reply:
idk I did it like this to avoid code duplication since 99% are similar

Reply:
You forgot to change the title of the dialog - adding that if statement imo will make it more worth it to make 2 functions with hard-coded values